app/web/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -72,6 +72,153 @@ const StyledHelpTextContainer = styled("div")(({ theme }) => ({
   marginBottom: theme.spacing(2),
 }));
 
+function ResponseGuideLinks({
+  isPast,
+  isHost,
+  status,
+}: {
+  isPast: boolean;
+  isHost: boolean;
+  status: HostRequestStatus;
+}) {
+  const isHostPending =
+    !isPast &&
+    isHost &&
+    status === HostRequestStatus.HOST_REQUEST_STATUS_PENDING;
+
+  const isSurferRejected =
+    !isHost && status === HostRequestStatus.HOST_REQUEST_STATUS_REJECTED;
+
+  return (
+    <>
+      {isHostPending && (
+        <StyledHelpTextContainer>
+          <Typography variant="body1">
+            <Trans i18nKey="messages:host_pending_request_help_text">
+              <StyledLink variant="body1" href={howToRespondRequestGuideUrl}>
+                Things to consider
+              </StyledLink>{" "}
+              before responding.
+            </Trans>
+          </Typography>
+        </StyledHelpTextContainer>
+      )}
+      {isSurferRejected && (
+        <StyledHelpTextContainer>
+          <Typography variant="body1">
+            <Trans i18nKey="messages:surfer_declined_request_help_text">
+              <StyledLink variant="body1" href={howToWriteRequestGuideUrl}>
+                Read our guide
+              </StyledLink>{" "}
+              on how to write a request that will get accepted.
+            </Trans>
+          </Typography>
+        </StyledHelpTextContainer>
+      )}
+    </>
+  );
+}
```

Comment:
You probably want a null case at the top, like:

```javascript
if (!isHostPending && !isSurferRejected) {
    return null;
```

Reply:
And done :)